Makefile

```diff
@@ -0,0 +1,18 @@
+all: build
+
+dependencies:
+	go get -u github.com/golang/lint/golint
+	go get -u golang.org/x/tools/cmd/cover
+	go get -u golang.org/x/tools/cmd/vet
+
+build:
+	go fmt  ./...
+	golint ./...
+	go vet  ./...
+	go test -v -bench=.
+	go install ./...
+	go test -covermode=atomic -coverprofile=cover.out . \
+	&& go tool cover -html=cover.out -o=cover.html \
+	&& rm cover.out
+
+.PHONY: all dependencies build
```

Comment:
this isn't a makefile, it's just a bash alias. There is literally nothing in here that is specific to this project. It could be run on any Go library.